logstash-core/spec/api/lib/api/support/resource_dsl_methods.rb

```diff
@@ -23,6 +23,21 @@ def test_api(expected, path)
       it "should respond OK" do
         expect(last_response).to be_ok
       end
+
+      
+      describe "the default metadata" do
+        it "should include the host" do
+          expect(payload["host"]).to eql(Socket.gethostname)
+        end
+
+        it "should include the version" do
+          expect(payload["version"]).to eql(LOGSTASH_CORE_VERSION)
+        end
+
+        it "should include the http address" do
+          expect(payload["http_address"]).to eql("#{Socket.gethostname}:9600")
```

Comment:
is there a way to not hardcode `9600` here in the `expect`. wondering if we can use the default constant which is defined in the core